rdf-abac-benchmark/src/main/java/io/telicent/jena/abac/labels/LabelsStoreRocksDBBenchmark.java

```diff
@@ -0,0 +1,100 @@
+package io.telicent.jena.abac.labels;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.jena.graph.NodeFactory;
+import org.apache.jena.graph.Triple;
+import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.infra.Blackhole;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+@BenchmarkMode(Mode.AverageTime) // Measures average execution time per operation
+@OutputTimeUnit(TimeUnit.MILLISECONDS) // Results in milliseconds
+@State(Scope.Thread) // Each thread gets its own instance
+public class LabelsStoreRocksDBBenchmark {
+
+    private static final int LABEL_LENGTH = 10;
```

Comment:
I think most of the labels we see tend to be quite a bit longer, I'm not sure how much that impacts on performance but maybe worth setting this to something a bit larger like `50` or `100` (or possibly a random size within that range, though I appreciate a fixed size label allows the benchmark to focus on one specific aspect of functionality)

Reply:
One thing we could do is to put the benchmark module in an off-by-default `<profile>` and then set up a `cron` triggered build that runs with that module on once a week and uploads the benchmark results as a build artifact, it could potentially even retrieve the previous weeks results and compare them to flag regressions.

But that's a whole ton of GitHub Actions "joy" to make work so definitely not for now!